test/kt/CMakeLists.txt

```diff
@@ -124,7 +124,49 @@ if (ONEDPL_TEST_ENABLE_KT_ESIMD)
     _generate_esimd_sort_test("256" "32" "double" "" 1000) # segfault
 endif()
 
+function (_generate_scan_test _data_per_work_item _work_group_size _type)
```

Comment:
Let's rename it to `_generate_igpu_scan_test` to highlight the KT backend.

Reply:
Fixed